### ISSUE TITLE
Document successfully deployed (but it's a lie)

### ISSUE THREAD

**Issue by csgillespie** [rstudio/rsconnect#108]
I've just tried to deploy a simply [app](https://github.com/rcourses/nclRshiny/blob/master/vignettes/components.Rmd). The deployment seemed to be successful:

```{r}
Loading required namespace: rmarkdown
Discovering document dependencies... OK
Preparing to deploy document...DONE
Uploading bundle for document: 85...DONE
Deploying bundle: 317 for document: 85 ...
Deploying to R version '3.2.4' (Local version is '3.3.0')
2016/06/06 10:44:20.896346072 Installed packrat SHA is 211179210c1d81db703973351f95d3c96dd557b7; packaged packrat SHA is 211179210c1d81db703973351f95d3c96dd557b7
2016/06/06 10:44:20.897805485 Packrat is up-to-date.
2016/06/06 10:44:20.897977357 R version: 3.2.4
<snip>

2016/06/06 10:44:49.998485179
2016/06/06 10:44:49.998498265 Output created: /opt/rstudio-connect/mnt/report/components.html
Document successfully deployed to https://bookdown.org/csgillespie/components/
```

But   https://bookdown.org/csgillespie/components/ returns a 404

**Comment by csgillespie** [comment]
Fixed issue. I think one of my Chrome plugins had block the authorisation page.

**Comment by trestletech** [comment]
Thanks for following up, Colin. If you're able to track down the specific source of the issue we'd love to hear it.

**Comment by csgillespie** [comment]
@trestletech I think I've found the issue. 

I just tried to deploy a new app. The first time I uploaded, I made a silly mistake and the upload failed. I opened the browser to view the [log](https://bookdown.org/connect/#/apps/98/logs/yHwpXge397FjKf42)

I fixed the mistake and re-deployed. The app was successful. However, since bookdown.org was already open I **didn't** get a new tab on my browser asking if I wish to make the app public. Hence the app was never made public.

Suggested fix. 

Rather than have

```
Document successfully deployed to ....
```
display in the R console. Remind users they need to make the app visible - perhaps point to a helpful url.
